frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -1143,6 +1138,16 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
                 actions.reportMessageTooLargeWarningSeen(values.sessionRecordingId)
             }
         },
+        currentPlayerState: (prev, next) => {
```

Comment:
pretty sure we're setting this more than we need to, but this at least can report on that too high a number